Lib/zipfile.py

```diff
@@ -1966,25 +2002,42 @@ def _compile(file, optimize=-1):
 
 
 def main(args=None):
+    """ Command line operation."""
     import argparse
 
+    zip_name = None
+    files = []
+    mode = None
+
     description = 'A simple command-line interface for zipfile module.'
-    parser = argparse.ArgumentParser(description=description)
-    group = parser.add_mutually_exclusive_group(required=True)
-    group.add_argument('-l', '--list', metavar='<zipfile>',
-                       help='Show listing of a zipfile')
-    group.add_argument('-e', '--extract', nargs=2,
-                       metavar=('<zipfile>', '<output_dir>'),
-                       help='Extract zipfile into target dir')
-    group.add_argument('-c', '--create', nargs='+',
-                       metavar=('<name>', '<file>'),
-                       help='Create zipfile from sources')
-    group.add_argument('-t', '--test', metavar='<zipfile>',
-                       help='Test if a zipfile is valid')
+    epilog = ' '.join([
+        'If a directory is passed as a parameter to create or append',
+        'it will be recursively added. Wildcards are supported, glob style,',
+        'and a wildcard such as **/*.py will recurse.  All recursion will',
+        'skip any directories starting with . such as .git, etc.'
```

Comment:
Help message in both tarfile and zipfile updated to include information on how to override, `.**/*` and that this is as for glob.iglob() which is used.